assets/scss/_footer.scss

```diff
@@ -0,0 +1,59 @@
+@use 'uswds-core' as *;
+
+$base-font-size: 1rem;
+$base-font-sm: ($base-font-size * 0.8125);
```

Comment:
Minor: Seems like we could use a [`font-body-2xs` design system utility class](https://designsystem.digital.gov/utilities/font-size-and-family/) to get a standard size close to this ad hoc sizing (`.81rem` vs. `.8125rem`). But also I'm not really sure the rationale behind having this small in the first place, and would almost rather just let it stay as normal size. Not like we're running out of space here.

Not a blocker since I assume we're just copying from `identity-site`, but I think the application here is a bit different since we don't have links in this region like we have on the brochure site.

Reply:
I'm happy to have this footer be the baseline. Let me get the ok from Product to change the size, so this doesn't get bounced back to me later.